accelerator/http.c

```diff
@@ -245,11 +223,19 @@ static int ta_http_header_iter(void *cls, enum MHD_ValueKind kind,
   return MHD_YES;
 }
 
-static int ta_http_handler(void *cls, struct MHD_Connection *connection,
-                           const char *url, const char *method,
-                           const char *version, const char *upload_data,
-                           size_t *upload_data_size, void **ptr) {
-  int ret = MHD_NO;
+static int print_req_log(void *cls, const struct sockaddr *addr, socklen_t addrlen) {
```

Comment:
Since it's already for logging, maybe rename this to `request_log` or `req_log`?

Reply:
request_log is nice!